podium-gateway/src/main/webapp/app/backoffice/modules/organisation/organisation-form/organisation-form.component.ts

```diff
@@ -113,9 +113,8 @@ export class OrganisationFormComponent implements OnInit {
         }
     }
 
-    isOrganisationAdmin (currentUser: User) {
-        if (currentUser) {
-            return currentUser.authorities.indexOf('ROLE_ORGANISATION_ADMIN') > -1;
-        }
+    get canActivateOrganisation() {
+        let authorities = this.currentAccount?.authorities || [];
+        return authorities.includes('ROLE_PODIUM_ADMIN') || authorities.includes('ROLE_BBMRI_ADMIN');
```

Comment:
The Podium admin plays no role in organisation administration.
```suggestion
        return authorities.includes('ROLE_BBMRI_ADMIN');
```

Reply:
That's what I thought as well, but since the Podium admin role was allowed to toggle organisation active/inactive state on the organisation list page I assumed it was intentional.
I'll change the organisation.component.html then.